### DISCUSSION TITLE
DNN to Oqtane

### DISCUSSION THREAD

**Discussion by wbonekamp** [oqtane/oqtane.framework#1768] [General]
Is there a migration path for DNN to Oqtane.

We really like to convert some DNN website to Oqtane... 
Is this already possible or is it too early?
Which DNN vendors are already active/available for Oqtane
Where can we find modules/extensions like easydnn news, action form, open content?

**Comment by sbwalker** [comment]
@wbonekamp there is currently no automated upgrade path from DNN to Oqtane. In theory it would not be difficult to create a data migration for the core entities, as the underlying data model is very similar. However any custom modules created for DNN would need to be rewritten for Oqtane as the framework architectures are completely different. At this point there are very few DNN commercial module vendors who have migrated their modules to Oqtane... however I am sure they may consider it if there is enough demand from customers/users.
